client/reader/following-manage/empty.jsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from 'components/empty-content';
+import { recordAction, recordGaEvent, recordTrack } from 'reader/stats';
+
+class FollowingManageEmptyContent extends React.Component {
+	componentDidMount() {
+		recordTrack( 'calypso_reader_empty_manage_following_loaded' );
+	}
+
+	recordAction = () => {
+		recordAction( 'clicked_discover_on_empty_manage_following' );
+		recordGaEvent( 'Clicked Discover on EmptyContent in Manage Following' );
+		recordTrack( 'calypso_reader_discover_on_empty_manage_following_clicked' );
+	};
+
+	render() {
+		/* eslint-disable wpcalypso/jsx-classname-namespace */
+		const action = (
+			<a
+				className="empty-content__action button is-primary"
+				onClick={ this.recordAction }
+				href="/discover"
+			>
+				{ this.props.translate( 'Explore Discover' ) }
+			</a>
+		);
+		/* eslint-enable wpcalypso/jsx-classname-namespace */
+
+		return (
+			<EmptyContent
+				action={ action }
+				title={ this.props.translate( "You haven't followed any sites yet" ) }
+				line={ this.props.translate( 'Search for a site above or explore Discover.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Search for a site or explore Discover.' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).